validator/remote/src/main/java/tech/pegasys/teku/validator/remote/BeaconNodeReadinessManager.java

```diff
@@ -114,7 +114,9 @@ private SafeFuture<Void> performReadinessCheck(
         .getSyncingStatus()
         .thenApply(
             syncingStatus -> {
-              if (!syncingStatus.isSyncing() || syncingStatus.getIsOptimistic().orElse(false)) {
+              if (!syncingStatus.isElOffline().orElse(false)
```

Comment:
can we put this into a SyncStatus method like `syncingStatus.isReady()` (or move the condition calculation on private method here), just for make this more clear with this additional condition.

Reply:
Done